i18n4k-core/src/iosMain/kotlin/de.comahe.i18n4k/Locale.kt

```diff
@@ -0,0 +1,39 @@
+package de.comahe.i18n4k
+
+import platform.Foundation.NSLocale
+import platform.Foundation.countryCode
+import platform.Foundation.currentLocale
+import platform.Foundation.languageCode
+import platform.Foundation.variantCode
+
+actual data class Locale actual constructor(
+    val language: String,
+    val country: String,
+    val variant: String
+) {
+
+    actual constructor(
+        language: String
+    ) : this(language, "", "")
+
+    actual constructor(
+        language: String,
+        country: String
+    ) : this(language, country, "")
+
+    actual fun getLanguage(): String = language
+    actual fun getCountry(): String = country
+    actual fun getVariant(): String = variant
+
+    override fun toString(): String = toTag()
+}
+
+actual val systemLocale: Locale
```

Comment:
Probably we need to add a "windowsCommon" and a "linuxCommon" source set where `actual val systemLocale` is defined for these platforms . They could be set to "en" until they are correctly implements for these platforms.